Week-4/Assignment-Eight-Javascript-Fundamentals/javascript-arrays.js

```diff
@@ -0,0 +1,18 @@
+function divideArray(arr) {
+  const evenNums = [],
+    oddNums = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    if (arr[i] % 2 === 0) {
+      evenNums.push(arr[i]);
+    } else {
+      oddNums.push(arr[i]);
+    }
+  }
+
+  console.log("Even numbers: ", evenNums.length ? evenNums.sort() : "None");
```

Comment:
will this work ??

Reply:
Hi Harsh,
Thanks for the feedback. The sort function which I have written will only work for single-digit values. Let me update the PR with the correct code